### PR TITLE
fix Chinese translation about reserved-cpus

### LINKED ISSUE #24881: 为系统守护进程预留计算资源
English:
reserved-cpus is meant to define an explicit CPU set for OS system daemons and kubernetes system daemons

Chinese:
reserved-cpus 旨在为操作系统守护程序和 kubernetes 系统守护程序定义一个显式 cpuset

Actually, it does not create a directory in /sys/fs/cgroup/cpuset/ to keep the reserved CPUs in it. What CPU Manager do is to avoid allocating them to Pods.
So, it should be translated as 
reserved-cpus 旨在为操作系统守护程序和 kubernetes 系统守护程序预留一组明确指定编号的CPU

### PR DESCRIPTION
**Pull request by julyclyde**
fix #24881

### CODE REVIEW

**Comment by ydFu** [comment]
@julyclyde
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by julyclyde** [comment]
Already signed CLA minutes ago.

File: content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md [hunk 1]
```diff
@@ -256,8 +256,8 @@ If the Kubelet **does not** have `--system-reserved-cgroup` and `--kube-reserved
 the explicit cpuset provided by `reserved-cpus` will take precedence over the CPUs
 defined by `--kube-reserved` and `--system-reserved` options.
 -->
-`reserved-cpus` 旨在为操作系统守护程序和 kubernetes 系统守护程序定义一个显式 CPU
-集合。`reserved-cpus` 适用于不打算针对 cpuset 资源为操作系统守护程序和 kubernetes
+`reserved-cpus` 旨在为操作系统守护程序和 kubernetes 系统守护程序预留一组明确指定编号的
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md:259]: 原来的翻译中”定义“一词用的不好。
这里的”预留“也不太恰当。
是否可考虑使用”保留“？

**Pull request merged**